tests/chart_tests/test_elasticsearch.py

```diff
@@ -120,6 +120,30 @@ def test_elasticsearch_securitycontext_defaults(self, kube_version):
             assert pod_data["securityContext"]["runAsNonRoot"] is True
             assert pod_data["securityContext"]["runAsUser"] == 1000
 
+    def test_elasticsearch_client_securitycontext_overrides(self, kube_version):
+        """Test ElasticSearch client with securityContext custom values."""
+        docs = render_chart(
+            kube_version=kube_version,
+            values={
+                "elasticsearch": {
+                    "securityContext_esclient": {
+                        "capabilities": {"add": ["IPC_LOCK"]},
+                    },
+                    "securityContext": {
+                        "capabilities": {"add": ["SYS_RESOURCE"]},
+                    },
+                }
+            },
+            show_only=[
+                "charts/elasticsearch/templates/client/es-client-deployment.yaml",
+            ],
+        )
+        assert len(docs) == 1
+        doc = docs[0]
+        pod_data = doc["spec"]["template"]["spec"]["containers"][0]
+        assert pod_data["securityContext"]["capabilities"]["add"] == ["IPC_LOCK"]
+        assert pod_data["securityContext"]["capabilities"]["add"] != ["SYS_RESOURCE"]
```

Comment:
With the way this test is written, this line is redundant. The prior line checks the whole `pod_data["securityContext"]["capabilities"]["add"]` data structure to make sure it is equal to a list with one item whose value is `IPC_LOCK`. That inherently excludes `SYS_RESOURCE` from the list.
```suggestion
```